example/cloudexec.toml

```diff
@@ -4,11 +4,66 @@ timeout = "48h"
 
 [commands]
 setup = '''
+
+########################################
+## Required Configuration and Dependencies
+
+# set hostname
+echo "Setting hostname..."
+echo "cloudexec" >/etc/hostname
+hostname -F /etc/hostname
+
+echo "Installing prereqs..."
+export DEBIAN_FRONTEND=noninteractive
+apt-get update
+apt-get install -y jq s3cmd tmux python3-pip python3-venv unzip
+
+echo "Downloading doctl..."
+curl -fsSL -o /tmp/doctl-1.92.0-linux-amd64.tar.gz https://github.com/digitalocean/doctl/releases/download/v1.92.0/doctl-1.92.0-linux-amd64.tar.gz
+echo "Extracting doctl..."
+tar -xzf /tmp/doctl-1.92.0-linux-amd64.tar.gz -C /tmp
+echo "Installing doctl..."
+mv /tmp/doctl /usr/local/bin
+echo "Cleaning up..."
+rm /tmp/doctl-1.92.0-linux-amd64.tar.gz
```

Comment:
I think most everything above this line should be moved to user_data directly since its critical for cloudexec to work and we shouldn't rely on the user setting it up right https://github.com/crytic/cloudexec/blob/44e27e3f7a2051895cb422a94c946c4ec16b2e3a/cmd/cloudexec/user_data.sh.tmpl